src/Compilers/Extension/CompilerPackage.cs

```diff
@@ -110,7 +111,32 @@ private void WriteMSBuildFiles(string packagePath, string hiveName)
             }
         }
 
-        private static void WriteMSBuildFile(string content, string relativeFilePath)
+        private string GetMSBuildVersionString()
+        {
+            var dte = (DTE)GetService(typeof(SDTE));
+            var parts = dte.Version.Split('.');
+            if (parts.Length == 0)
+            {
+                throw GetBadVisualStudioVersionException(dte.Version);
+            }
+
+            switch (parts[0])
```

Comment:
MSBuild isn't guaranteed to version with Visual Studio.  It just happens to do so.  Want to make sure that when VS changes we verify that MSBuild change too.